### ISSUE TITLE
[BUG] Fail to enable diagnostic logs for App Service

### ISSUE THREAD

**Issue by Flanker32** [Azure/azure-libraries-for-java#759]
**Describe the bug**
Unable to enable diagnostic logs for App Service, tried both `withContainerLoggingEnabled()` and `updateDiagnosticLogsConfiguration()`.

***Exception or Stack Trace***
[Logs.txt](https://github.com/Azure/azure-libraries-for-java/files/3294931/Logs.txt)

**To Reproduce**
``` java
final Azure.Configurable configurable = Azure.configure()
                .withLogLevel(LogLevel.BODY_AND_HEADERS);
Azure azure = configurable.authenticate(AzureCliCredentials.create()).withDefaultSubscription();
WebApp webapp = azure.webApps().getByResourceGroup("RESOURCE_GROUP", "WEBAPP");
webapp.update().withContainerLoggingEnabled().apply();
webapp.restart();
```

**Expected behavior**
App service server log is enabled, could see the streaming log in portal

**Screenshots**
![image](https://user-images.githubusercontent.com/12445236/59573039-8b445d00-90e3-11e9-8c4e-c4e456fc53f8.png)

**Setup (please complete the following information):**
 - OS:  Win 10 1903
 - IDE : IntelliJ 2019.1
 - Version of the Library used Azure-SDK-For-Java/1.21.0

**Additional context**
When I try with the correspond rest API with the same request body in the log, **it works**, here is the request body
``` json
{
    "properties": {
        "applicationLogs": {
            "fileSystem": {
                "level": "Off"
            },
            "azureBlobStorage": {
                "level": "Off"
            }
        },
        "httpLogs": {
            "fileSystem": {
                "retentionInMb": 35,
                "retentionInDays": 0,
                "enabled": true
            }
        },
        "failedRequestsTracing": {
            "enabled": false
        },
        "detailedErrorMessages": {
            "enabled": false
        }
    }
}
```

Besides, I found the management SDKdo update the app service setting, I could update the `FileSystemQuotaInMB` and `FileSystemRetentionDays`, but the file system HTTP logs is still not enabled.

**Information Checklist**
Kindly make sure that you have added all the following information above and checkoff the required fields otherwise we will treat the issuer as an incomplete report
- [x] Bug Description Added
- [x] Repro Steps Added
- [x] Setup information Added

**Comment by praries880** [comment]
Thank you for opening this issue! We are routing it to the appropriate team for follow up.

@yaohaizh Can you kindly take a look at this?

**Comment by weidongxu-microsoft** [comment]
Could be caused by concurrency on service, at config/web and config/log

**Comment by weidongxu-microsoft** [comment]
To be released on 1.26
